easybuild/easyconfigs/c/CondaCreate/requirements.txt

```diff
@@ -0,0 +1 @@
+perl-app-cpanminus
```

Comment:
Its up to you. I wanted to include a sample of a requirements file, but all its just a line separated by line of conda packages to install.

If you prefer it could be an option supplied by EB_condaCreate

`conda_packages: ['perl-app-cpanminus', 'picard', 'samtools']` ?

I went with requirements.txt because it seems to be a convention of the python community, but I also like the idea of having the whole thing contained in the eb config.